gauntlet/packages/gauntlet-solana-contracts/src/core/proto/ocr2Proto.ts

```diff
@@ -0,0 +1,45 @@
+// Autogenerated from ocr2Proto.proto with Protobuf.toJSON()
+export const descriptor = {
```

Comment:
Autogenerated descriptor with https://github.com/smartcontractkit/chainlink-solana/blob/8bb813d26e1ab55e54fbf6b938c33417685c86cd/gauntlet/packages/gauntlet-solana-contracts/src/core/proto/ocr2Proto.proto as source
JSON descriptor is still used as TS transpilation does not allow to include `.proto` files, that will be needed for the binary

Reply:
Let's add a `yarn` script that will do the auto-generation and write the descriptor to the file so there are no manual steps involved.